Sources/Screens/Tabs/HomeViewController.swift

```diff
@@ -54,6 +57,14 @@ extension HomeViewController {
         }
         return cell
     }
+    
+    override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        didTapCard()
```

Comment:
Is there a reason for not doing `viewModel.cardTapped()` here directly rather than wrapping it in the `didTapCard` function and then calling that?

Reply:
Not really 😅 I can change it